spec/controllers/finders_controller_spec.rb

```diff
@@ -108,22 +108,21 @@
           "suggested_queries": []
         }|
 
-        stub_request(:get, "#{Plek.current.find('search')}/search.json")
+        stub = stub_request(:get, "#{Plek.current.find('search')}/search.json")
           .with(
             query: {
               count: 10,
               fields: "title,link,description,public_timestamp,popularity,content_purpose_supergroup,format,walk_type,place_of_origin,date_of_introduction,creator",
               filter_document_type: "mosw_report",
-              order: "-closing_date",
+              order: "-public_timestamp",
               start: 0
             }
           )
           .to_return(status: 200, body: rummager_response, headers: {})
-      end
 
-      it "returns a 404 when requesting an atom feed, rather than a 500" do
```

Comment:
It might be nice to retain a test for what happens when one requests a finder and it doesn't exist.

Reply:
Do you mean if the content item doesn't exist?

Reply:
Ah my bad. The 404 wouldn't reach this application if an atom feed route wasn't published.